### PR TITLE
Fix attention mask

### PR DESCRIPTION
**Pull request by o-alexandre-felipe**
Fix the WARNING
> The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

The fix was very simple, passing `return_attention_masks=True` also when `model.can_generate()`

I also wrapped `MODEL_IDs` in the bash scripts to make it readable without scrolling horizontally.

### CODE REVIEW

**Comment by Vaibhavs10** [comment]
Closing this since this is fixed in main.
